slack/slack.asm

```diff
@@ -26,3 +26,9 @@ SECTION "slack.asm@Corrupted Sprites", ROMX
 
 CorruptedSpriteGFX:
 INCBIN "slack/sprites_corrupted.2bpp"
+
+SECTION "slack.asm@Unknowne7e9f", ROMX
+
+Unknowne7e9f:
+INCBIN "slack/unknown_e7e9f.bin" ; Loaded as corrupted graphics 
```

Comment:
Is this definitely just garbage in the slack space, or might it be loading real code as graphics?

Reply:
So far I have seen this slack space loaded as graphics during the intro
I haven't checked if this is loaded or accessed anywhere else

Reply:
If it has that "rectangular TV static" pattern to it, it's garbage.

Reply:
![image](https://user-images.githubusercontent.com/13794376/101783596-0b0e8480-3b2d-11eb-9608-80fd7600f454.png)

Tile editor shows this, and yes it also got loaded into VRAM